### ISSUE TITLE
Counter tied to a key with a reset

### ISSUE THREAD

**Issue by jahnjo** [runelite/runelite#12542]
**Is your feature request related to a problem? Please describe.**
I'm thinking of this feature purely for alchemical hydra, idk if a built in hydra attack counter would be against the rules, but I am always forgetting what count I am on with the hydra attacks and at the very least I would love to be able to hit a button to count the attacks and then have it reset when it hits 3 or something like that.

**Describe the solution you'd like**
Counter that increments when I hit a key, and when it reaches 3, reset. Or have settings to configure the counter like that.

**Describe alternatives you've considered**
Just count hydra attacks as a hydra plugin (not sure if legal or not)

**Additional context**

**Comment by ErikHumphrey** [comment]
Plugins that make the Alchemical Hydra fight easier are (not currently being considered](https://github.com/runelite/runelite/wiki/Rejected-or-Rolled-Back-Features).

**Comment by Nightfirecat** [comment]
We're not planning on making anything specific for the hydra fight. That said, some kind of counter plugin which has only manual triggers (such as a hotkey) would probably be fine. I'd envision this would look something like the plugin hub's [Zulrah helper plugin](https://github.com/while-loop/runelite-plugins/tree/zulrah-helper) which only responds to the user hitting hotkeys, not to ingame state.

**Comment by jahnjo** [comment]
Yeah I thought it would a problem, but this solution would be an external plugin that would be useful for hydra but not exclusively for hydra.

**Comment by jahnjo** [comment]
> We're not planning on making anything specific for the hydra fight. That said, some kind of counter plugin which has only manual triggers (such as a hotkey) would probably be fine. I'd envision this would look something like the plugin hub's [Zulrah helper plugin](https://github.com/while-loop/runelite-plugins/tree/zulrah-helper) which only responds to the user hitting hotkeys, not to ingame state.

Yeah exactly what I am thinking. I see attack, i hit 'j', counter goes from 0 to 1, and then when counter is at 2 and i hit 'j' counter resets to 0. But I can configure the counter to go on forever, or stop at a certain number, reset or dont reset, and which keybind to use.

**Comment by Hydrox6** [comment]
Personally, I'd say that the resetting shouldn't be automatic; make it trade the counting for quick maths on the current number.

**Comment by jahnjo** [comment]
Not sure what you mean by that.
